piccolo/columns/column_types.py

```diff
@@ -442,9 +454,23 @@ def __init__(
         **kwargs: Unpack[ColumnKwargs],
     ) -> None:
         self._validate_default(default, (str, None))
+
         self.default = default
         super().__init__(default=default, **kwargs)
 
+    def get_default_value(self):
+        """
+        MySQL does not allow unquoted TEXT literals in the DEFAULT
+        clause, so we use the expression in parentheses.
+        Only works in CREATE TABLE. MySQL does not allow default
+        values for TEXT columns in ALTER statements.
+        """
+        engine_type = self._meta.engine_type
+
+        if engine_type == "mysql":
+            return QueryString(f"('{self.default}')")
```

Comment:
I think this could be problematic.

`self.default` could be all sorts of different things - like `None` or a function, so if we embed it in a string we might end up with invalid DDL.

What if this logic lives in `get_sql_value` instead?

Reply:
But we need to somehow change the default value because `TEXT`, `BLOB`, and `JSON` do not allow unquoted literals in default . This is not allowed.
```sql
CREATE TABLE example (
    description TEXT DEFAULT 'hello'
);
```
We need to use an expression in parentheses.
```sql
CREATE TABLE example (
    description TEXT DEFAULT ('hello') -- expression
);
```
Only works in `CREATE TABLE` because MySQL does not allow default values ​​(at all, with or without expressions) for `TEXT`, `BLOB`, and `JSON`  columns in `ALTER` statements